docs/source/documentation/case-booking.html.md.erb

```diff
@@ -0,0 +1,32 @@
+---
+title: Case Bookings
+---
+
+# Case Bookings
+Cases are created at the contact stage of a users journey. When the user selects `Submit details`, the full payload is sent to the 
+CLA backend endpoint which via `post_case`. This sends the full payload to backend with all required information including information 
+provided on the contact us form.
+
+### Case reference
+The case reference is provided when the endpoint is called. This is then updated in the session to be provided on the confirmation page 
+by:
+
+```python
+session["case_reference"] = cla_backend.post_case(payload=payload)["reference"]
+```
+
+This in turn also posts the case to CLA Backend.
+
+### Reason for contacting
+When a user goes directly to Contact Us via the Reasons for Contacting form, the payload only creates a case with a 
+case number and the selections on the reasons for contacting form.
```

Comment:
is case number here actually a case reference? if so lets be consistent. If it isnt can we calrify the difference between a case number and a case reference.